public/docs/ts/latest/guide/index.jade

```diff
@@ -30,85 +22,68 @@ table(width="100%")
+  Each page includes code snippets that you can reuse in your applications.
+  These snippets are excerpts from a sample application that accompanies the page.
 
-  1. [Dependency Injection](dependency-injection.html) is the way to build large, maintainable applications
-  from small, single-purpose parts.
+block example-links
+  :marked
+    Look for a link to a running version of that sample near the top of each page,
+    such as this <live-example name="architecture"></live-example> from the [Architecture](architecture.html) page.
 
-  1. [Template Syntax](template-syntax.html) is a comprehensive study of Angular template HTML.
+    The link launches a browser-based, code editor where you can inspect, modify, save, and download the code.
 
-  After reading the above sections, you can skip to any other pages on this site.
+:marked
+  ## Reference pages
 
-  ## Code samples
+  The [Cheat Sheet](cheatsheet.html) lists Angular syntax for common scenarios.
 
-  Each page includes code snippets that you can reuse in your applications.
-  These snippets are excerpts from a sample application that accompanies the page.
+  The [Glossary](glossary.html) defines terms that Angular developers should know.
 
-  Look for a link to a running version of that sample near the top of each page,
-  such as this <live-example name="architecture"></live-example> from the [Architecture](architecture.html) page.
-  <p if-docs="ts">
-  The link launches a browser-based code editor where you can inspect, modify, save, and download the code.
-  </p>
+  The [Change Log](change-log.html) announces what's new and changed in the documentation..
 
-  A few early pages are written as tutorials and are clearly marked as such.
-  The rest of the pages highlight key points in code rather than explain each step necessary to build the sample.
-  You can always get the full source through the #{_liveLink}.
+  The [API Reference](../api/) is the authority on every public-facing member of the Angular libraries.
 
-  ## Reference pages
+  ## Feedback
 
-  - The [Cheat Sheet](cheatsheet.html) lists Angular syntax for common scenarios.
-  - The [Glossary](glossary.html) defines terms that Angular developers should know.
-  - The [API Reference](../api/) is the authority on every public-facing member of the Angular libraries.
+  We welcome feedback! 
 
-  ## We welcome feedback! 
+  Use the <a href="https://github.com/angular/angular.io" target="_blank" title="angular docs on github">angular.io Github repository</a> for **documentation** issues and pull requests.
 
-  - Use the [website GitHub repo](!{_ngDocRepoURL}) for **documentation** issues and pull requests.
-  - Use the [Angular GitHub repo](!{_ngRepoURL}) to report issues with **Angular** itself.
+  Use the <a href="https://github.com/angular/angular" target="_blank" title="angular source on github">Angular Github repository</a> to report issues with **Angular** itself.
```

Comment:
The two feedback links will need to be reverted to `!{_ngDocRepoURL}` and `!{_ngRepoURL}`, respectively; otherwise the links will be invalid for Dart.